README.md

```diff
@@ -40,7 +40,7 @@ This controller watches for changes on `Policies` in the cluster namespace on th
 ## Geting started
```

Comment:
This might need to be updated upstream...
```suggestion
## Getting started
```

Reply:
I included it in an additional commit in this PR:
https://github.com/open-cluster-management-io/governance-policy-framework-addon/pull/3